libraries/AC_Planck/AC_Planck.h

```diff
@@ -66,8 +66,33 @@ class AC_Planck {
   //Get a position, velocity, yaw command
   bool get_posvel_cmd(Location &loc, Vector3f &vel_cms, float &yaw_cd, bool &is_yaw_rate);
 
+  // handle ack/nack message from ACE
+  void handle_planck_ack(const mavlink_message_t &msg);
+
   uint32_t mux_rates(float rate_up,  float rate_down);
 
+  //Returns true if the last command req was actively NACKd or timed out
+  bool was_last_request_rejected() {
+    if(_ack_status == PLANCK_WAITING_FOR_ACK && ((AP_HAL::millis() - _last_cmd_req_t_ms) > 500)) {
```

Comment:
Waiting for ack is distinct from being rejected. This is checking if 
1. the _ack_status is PLANCK_WAITING_FOR_ACK
AND
2. It has been > 500 ms since the last command request

waiting_for_ack() will return the last command req id, IF  
1 _ack_status is PLANCK_WAITING_FOR_ACK
AND
2. It has been <= 500 ms since the last command request,

otherwise it returns  -1 not 0